docs/how_to/custom_plugins.rst

```diff
@@ -297,7 +297,7 @@ clause.
     ``twittersearch``, ``twitterrecententries`` and ``video``.
 
     Additionally, it is *recommended* that you avoid using ``page`` as a model
```

Comment:
This is great, however, it can be a little confusing as it sorta reads like you shouldn't have a PageField in your model. I would recommend language something like:

... it is recommended that you avoid declaring a field with the name "page" in a subclass of CMSPlugin ...